### ISSUE TITLE
Parse new RRISD gradebook's data

### ISSUE THREAD

**Issue by tristanseifert** [quickhac/qhac-ios#50]
It's horribly broken and doesn't work, because RRISD's new gradebook is a piece of shit.

**Comment by tristanseifert** [comment]
This needs quite a bit of backend work to be done. We need to retool the code to deal with the fact that one request gets us all the six weeks grade, as well as all assignments—but for all classes in a given six-weeks period.

Class discovery needs to be significantly changed to go through all marking periods before deciding which classes are available. Determination of which classes actually have data is simplified to checking for rows in its table.

Basically, RRISD's gradebook sucks.
